tools/rdm/TestMixins.py

```diff
@@ -775,7 +775,7 @@ def Test(self):
 
   def GetMinLevel(self):
     min_level_decreasing = self.MinLevelDecreasing()
-    if self.Property('split_levels_supported'):
+    if not self.Property('split_levels_supported'):
```

Comment:
It looks like it matches the spec now. Would it be clearer if you just used a ternary operator for the field on the AckGetResult a few lines down though?